src/Core/Utilities/StrictEmailAddressAttribute.cs

```diff
@@ -25,7 +26,12 @@ public override bool IsValid(object value)
                     return false;
                 }
             }
-            catch (ParseException e)
+            catch (ParseException)
+            {
+                return false;
+            }
+
+            if (!Regex.IsMatch(emailAddress, @"@.+\."))
```

Comment:
Please modify to `@.+\.[^$]` so as not to include `someone@somewhere.` as a valid email address (must not end in a dot); or perhaps more appropriate to use `@.+\.[\S\w]` to it ends in a word character, but not sure if you can end with numbers, etc.; so basically let's just prevent the tail end having an illegal character.

Reply:
good catch!

Reply:
It looks like we should limit to [A-Za-z0-9] as ending characters: https://en.wikipedia.org/wiki/Hostname#Syntax